server/src/test/java/MembershipTest.java

```diff
@@ -1,8 +1,27 @@
 import static org.junit.jupiter.api.Assertions.*;
 import org.junit.jupiter.api.Test;
+import com.example.model.Perk;
+import com.example.model.Membership;
+import com.example.model.User;
 
 import java.util.ArrayList;
 
 class MembershipTest {
 
+    @Test
+    public void MembershipTest(){
```

Comment:
can you move this into the test/java/com/example/model package

and make sure they are packaged `package com.example.model;
`

Reply:
still need to move the membershiptest and product test to the correct files